client-src/elements/chromedash-featurelist.ts

```diff
@@ -374,7 +370,7 @@ class ChromedashFeaturelist extends LitElement {
     let filteredWithState = this.filtered.map(feature => {
       const editable =
         this.isSiteEditor ||
-        (this.editableFeatures && this.editableFeatures.includes(feature.id));
+        (this.editableFeatures && this.editableFeatures.includes(feature));
```

Comment:
Is there a reason to change checking `feature` vs. `feature.id` here?

Reply:
id does not exist on the type, so I assumed that it was initially a typo

Reply:
but according to the updated type we should indeed include .id. my bad